client/landing/gutenboarding/components/locale-context/index.tsx

```diff
@@ -47,13 +47,20 @@ const ChangeLocaleContext = React.createContext< ChangeLocaleFunction >( () => {
 export const ChangeLocaleContextConsumer = ChangeLocaleContext.Consumer;
 
 export const LocaleContext: React.FunctionComponent = ( { children } ) => {
-	const [ contextLocaleData, setContextLocaleData ] = React.useState< LocaleData | undefined >();
+	const [ contextLocaleData, setContextLocaleData ] = React.useState< LocaleData | undefined >(
+		// Call loadInitialLocale() ASAP, but wrapped to ignore its Promise
+		() => {
+			loadInitalLocale();
+		}
+	);
+	const [ localeDataLoaded, setLocaleDataLoaded ] = React.useState< true | null >( null );
```

Comment:
Why not use a boolean?

```suggestion
	const [ localeDataLoaded, setLocaleDataLoaded ] = React.useState( false );
```

Reply:
Oh, it's because you "render" the `null` when it's falsey 👍

Reply:
This jumped out to me, too. Would it be clearer to have this set to a boolean with `false` as default, then in 92 use a ternary instead of `&&` to render `null` so that rendering `null` is more explicit? This could just be my style preference, though!